jvm-execution/src/main/scala/com/avast/server/toolkit/execution/LoggingUncaughtExceptionHandler.scala

```diff
@@ -0,0 +1,11 @@
+package com.avast.server.toolkit.execution
+
+import java.lang.Thread.UncaughtExceptionHandler
+
+import com.typesafe.scalalogging.LazyLogging
+
+object LoggingUncaughtExceptionHandler extends UncaughtExceptionHandler with LazyLogging {
+
+  override def uncaughtException(t: Thread, ex: Throwable): Unit = logger.error(s"Uncaught exception on thread ${t.getName}", ex)
```

Comment:
Is it actually worth it to include scala-logging just because of single log message?